### PR TITLE
add `elastic/apm-mutating-webhook` to the build

### PR DESCRIPTION
**Pull request by bmorelli25**
For https://github.com/elastic/apm-mutating-webhook/issues/37.

### CODE REVIEW

**Comment by bmorelli25** [comment]
Sorry. Converting to draft as I didn't initially realize this would be blocked by https://github.com/elastic/apm-mutating-webhook/pull/46.

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

File: conf.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 repos:
     apm-aws-lambda:       https://github.com/elastic/apm-aws-lambda.git
+    apm-mutating-webhook: https://github.com/elastic/apm-mutating-webhook
```
Comment (bmorelli25) [inline conf.yaml:3]:
```suggestion
    apm-mutating-webhook: https://github.com/elastic/apm-mutating-webhook.git
```

**Review by gtback — APPROVED** [review]
LGTM!

**Pull request merged**